feature-libs/tracking/personalization/root/personalization-root.module.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { TestBed } from '@angular/core/testing';
+import { CmsConfig } from '@spartacus/core';
+import {
+  defaultPersonalizationComponentsConfig,
+  PersonalizationRootModule,
+} from './personalization-root.module';
+import { PERSONALIZATION_FEATURE } from './feature-name';
+
+const MockCmsConfig: CmsConfig = {
+  featureModules: {
+    [PERSONALIZATION_FEATURE]: {
+      cmsComponents: ['PersonalizationScriptComponent'],
+    },
+  },
+};
+
+describe('FeatureModule', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [PersonalizationRootModule],
+    });
+  });
+
+  it('initializes', () => {
+    const module = TestBed.inject(PersonalizationRootModule);
+    expect(module).toBeTruthy();
+  });
+
+  it('defaultPersonalizationComponentsConfig', () => {
+    const result = defaultPersonalizationComponentsConfig();
+    expect(result).toEqual(MockCmsConfig);
+  });
+});
```

Comment:
Are changes in this file related to this PR?

Reply:
At first glance, I thought this was omitted file during one of the merges from develop, but then I reminded myself that this was an effort to bump unit tests threshold on pipeline done by @mszczepaniak